### DISCUSSION TITLE
请问一下BT PT等下载怎么进行流量规避？

### DISCUSSION THREAD

**Discussion by szshingo** [coolsnowwolf/lede#9692] [R68S / 电犀牛专区]
如题！
光猫-软路由- AC- AP，要在哪里加入规则，进行流量规避，谢谢！
光猫ip:192.168.1.1 软路由：192.168.1.10 AC：192.168.1.5  NAS：192.168.0.51

在哪里加规则，哪几个ip要加进去，谢谢！

**Comment by Cathgao** [comment]
如果你问的是出国，只代理常用端口不就行了

**Reply by szshingo** [reply]
对的，就是出国，你的意思是，只代理要出国的设备的ip吗？这个好像可以，但openclash在哪里加？我是小白，我可能要研究一下，麻烦教教我在哪里设置，谢谢

**Reply by puteulanus** [reply]
<img width="569" alt="图片" src="https://user-images.githubusercontent.com/4849177/177197791-3cf32dcb-da45-4b4a-bbd7-419aeba58b6d.png">

**Reply by szshingo** [reply]
好的，太谢谢你了
